app/views/passwords/new.html.erb

```diff
@@ -1,19 +1,19 @@
 <!--Descriptive page name, messages and instructions-->
 <h3 class="heading"><%= ts("Forgotten your password?") %></h3>
 <p>
-  <%= ts("If you've forgotten your password, we can send you a temporary one.") %> 
+  <%= ts("If you've forgotten your password, we can send you a temporary one.") %>
   <%= ts("Please tell us the user name or email address you used when you signed up for your Archive account.") %>
 </p>
 <p>
-  <%= ts("If the temporary password doesn't work, please visit <a href=\"http://archiveofourown.org/admin_posts/1277\">Problems When Logging In</a> for help.".html_safe) %>
+  <%= ts("If the temporary password doesn't work, please visit %{problems_when_logging_in} for help.", :problems_when_logging_in => link_to(ts("Problems When Logging In"), admin_posts_path("1277"))) %>
```

Comment:
Unfortunately, this currently produces the following on the forgot password page: `If the temporary password doesn't work, please visit <a href="/admin_posts.1277">Problems When Logging In</a> for help.`

However, we can fix it! First, it needs an `.html_safe` at the end, after all the parentheses, so the HTML isn't escaped:

`<%= ts("If the temporary password doesn't work, please visit %{problems_when_logging_in} for help.", :problems_when_logging_in => link_to(ts("Problems When Logging In"), admin_posts_path("1277"))).html_safe %>`

Then the helper should be `admin_post_path("1277")` rather than `posts` -- when there's an `s`, it will generally point to an index, whereas the singular will point to an individual post (or work, or bookmark, etc):

`<%= ts("If the temporary password doesn't work, please visit %{problems_when_logging_in} for help.", :problems_when_logging_in => link_to(ts("Problems When Logging In"), admin_post_path("1277"))).html_safe %>`

And you can also use JSON style rather than the old hash rocket here and do `problems_when_logging_in: link_to` etc.

This page doesn't get touched very often, so if you're game for making a few more tweaks, it would be excellent if you had time to:
- combine the two translation strings in the first paragraph (one `ts()` per paragraph is generally good)
- change the few instances of single quotes to double
- remove the colon from the form label

If not, though, no worries!